### PR TITLE
Fix 1091 and other minor fixes

### LINKED ISSUE #1091: Menu system - btrfs subvolume issue
When trying to edit the subvolumes it deletes the item below the selected one.  In this case I was testing with 2 drives, I selected the boot drive, btrfs, suggested and then tried to remove the home subvolume, but instead the log was removed.  The idea being home would be mounted on the second drive.

before

after

### PR DESCRIPTION
**Pull request by svartkanin**
The things in this PR:
- Fixes #1091
- Only display btrfs option in the partitioning selection if there's actually a btrfs partition present
- Make menus cancellable

### CODE REVIEW

File: archinstall/lib/user_interaction/disk_conf.py [hunk 1]
```diff
@@ -41,13 +41,13 @@ def select_disk_layout(block_devices: list, advanced_options=False) -> Dict[str,
 	custome_mode = str(_('Select what to do with each individual drive (followed by partition usage)'))
 	modes = [wipe_mode, custome_mode]
 
-	print(modes)
-	mode = Menu(_('Select what you wish to do with the selected block devices'), modes, skip=False).run()
+	mode = Menu(_('Select what you wish to do with the selected block devices'), modes).run()
```
Comment (svartkanin) [inline archinstall/lib/user_interaction/disk_conf.py:44]: Remove the skip to make it cancellable

File: archinstall/lib/user_interaction/partitioning_conf.py [hunk 2]
```diff
@@ -165,7 +165,10 @@ def manage_new_and_existing_partitions(block_device: 'BlockDevice') -> Dict[str,
 			# 	# https://www.gnu.org/software/parted/manual/html_node/mklabel.html
 			# 	name = input("Enter a desired name for the partition: ").strip()
 
-			fstype = Menu(_('Enter a desired filesystem type for the partition'), fs_types(), skip=False).run()
+			fstype = Menu(_('Enter a desired filesystem type for the partition'), fs_types()).run()
```
Comment (svartkanin) [inline archinstall/lib/user_interaction/partitioning_conf.py:168]: Removed the skip to make it cancellable

**Comment by Torxed** [comment]
:partying_face: :tada:

**Comment by svartkanin** [comment]
@Torxed haha sorry I just pushed a fix for #1118 this very second, but you were quicker with merging :D

**Comment by svartkanin** [comment]
Hmm actually maybe it still made it into the merge

**Comment by Torxed** [comment]
Looks like it made it :) But it didn't auto-close the issue. But I can close it manually :)

**Pull request merged**